lib/util.ex

```diff
@@ -20,10 +20,16 @@ defmodule Util do
         tourneys = 1..10
                    |> Enum.map(&get_tourney_page(auth, &1))
                    |> List.flatten
-        {:ok, tourneys}
+        {:ok, %{tournament_data: tourneys, metadata: generate_tournaments_metadata()}}
     end
   end
 
+  def generate_tournaments_metadata do
```

Comment:
Until we have other fields to add (and I can't really think of any that would be useful at the moment), I'd suggest inlining this function.